app/models/host/base.rb

```diff
@@ -115,6 +115,7 @@ def dup
       :domain=, :domain_id=, :domain_name=, :to => :primary_interface
 
     attr_writer :updated_virtuals
+
```

Comment:
leftover

Reply:
Fixed.

Reply:
This was suggested by rubocop. Would you suggest I leave it there or remove it?